migrations/20180507182549_create_hxl_tags_tables.php

```diff
@@ -0,0 +1,697 @@
+				'respondee',
+				'population',
+				'affected',
+				'inneed',
+				'targeted',
+				'reached',
+			],
+			'attributes' => [
+				'f',
+				'm',
+				'i',
+				'infants',
+				'children',
+				'adolescents',
+				'adults',
+				'elderly',
+				'start',
+				'end',
+				'reported',
+				'event',
+				'killed',
+				'injured',
+				'infected',
+				'displaced',
+				'idps',
+				'refugees',
+				'abducted',
+				'threatened',
+			]
+		],
+		'description' => [
+			'tags' => [
+				'status',
+				'description',
+				'meta',
+				'beneficiary',
+				'item',
+				'need',
+				'service',
+				'impact',
+				'loc',
+				'region',
+				'adm1',
+				'adm2',
+				'adm3',
+				'adm4',
+				'adm5',
+				'country',
+				'org',
+				'contact',
+				'sector',
+				'subsector',
+				'activity',
+				'output',
+				'frequency',
+				'capacity',
+				'access',
+				'operations',
+				'value',
+				'currency',
+				'modality',
+				'channel',
+				'crisis',
+				'event',
+				'group',
+				'cause',
+				'severity',
+				'indicator',
+				'respondee',
+				'population',
+				'affected',
+				'inneed',
+				'targeted',
+				'reached',
+			],
+			'attributes' => [
+				'f',
+				'm',
+				'i',
+				'infants',
+				'children',
+				'adolescents',
+				'adults',
+				'elderly',
+				'start',
+				'end',
+				'reported',
+				'event',
+				'killed',
+				'injured',
+				'infected',
+				'displaced',
+				'idps',
+				'refugees',
+				'abducted',
+				'threatened',
+			]
+		],
+	];
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * http://docs.phinx.org/en/latest/migrations.html#the-abstractmigration-class
+     *
+     * The following commands can be used in this method and Phinx will
+     * automatically reverse them when rolling back:
+     *
+     *    createTable
+     *    renameTable
+     *    addColumn
+     *    renameColumn
+     *    addIndex
+     *    addForeignKey
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function up()
+    {
+
+		$this->table('hxl_tags')
+			->addColumn('tag_name', 'string', [
+				'null' => false,
+				'default' => false,
+				'comment' => 'The hxl tag. Examples: #geo, #population, #gender'
+			])
+			->addColumn('description', 'string', [
+				'null' => false,
+				'default' => false,
+				'comment' => 'The hxl tag description.'
+			])
+			->addIndex(['tag_name'], ['unique' => true])
+			->create();
+
+
+		$this->table('hxl_attributes')
+			->addColumn('attribute', 'string', [
+				'null' => false,
+				'default' => false,
+				'comment' => 'The hxl attribute. Examples: +f, +m, +adolescents'
+			])
+			->addColumn('description', 'string', [
+				'null' => false,
+				'default' => false
+			])
+			->addIndex(['attribute'], ['unique' => true])
+			->create();
+
+		$this->table('hxl_tag_attributes')
```

Comment:
Probably disable the primary key on this table, since the tag/attribute combos are unique anyway